### PR TITLE
VaHover docs

### LINKED ISSUE #322: Hover docs

### PR DESCRIPTION
**Pull request by SergeiPleshkov**
Closes #322 
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Improvement/refactoring (non-breaking change that doesn't add any feature but make things better)

### CODE REVIEW

File: packages/docs/examples/va-hover/Slot.vue [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<template>
+  <va-hover v-slot:default="{ hover }">
+    <div style="background-color: #babfc2;">
```
Comment (asvae) [inline packages/docs/examples/va-hover/Slot.vue:3]: `class="background--primary"` or something similar would probably make more sense.

File: packages/docs/locales/en/en.json [hunk 2]
```diff
@@ -728,13 +740,36 @@
...
+      },
+      "disabled": {
+        "title": "Disabled"
+      }
+    },
+    "faq": {
+      "questions": [
+        {
+          "question": "Can I wrap multiple elements with `va-hover`?",
+          "answer": "No, you should use exactly one element. If you use multiple, hover will work only for the first one."
```
Comment (asvae) [inline packages/docs/locales/en/en.json:770]: Any reason to have this limitation in the first place?

File: packages/docs/examples/va-hover/Disabled.vue [hunk 3]
```diff
@@ -0,0 +1,20 @@
+<template>
+  <div>
+    <va-hover disabled>
+      <div style="background-color: #babfc2;">
+        {{ value }}
+      </div>
+    </va-hover>
+    <input type="checkbox" v-model="value" />
```
Comment (asvae) [inline packages/docs/examples/va-hover/Disabled.vue:8]: What is that input for?

File: packages/docs/examples/va-hover/VModel.vue [hunk 4]
```diff
@@ -0,0 +1,20 @@
+<template>
+  <div>
+    <va-hover v-model="value">
+      <div style="background-color: #babfc2;">
+        {{ value }}
+      </div>
+    </va-hover>
+    <input type="checkbox" v-model="value" />
```
Comment (asvae) [inline packages/docs/examples/va-hover/VModel.vue:8]: No native components in docs.

File: packages/docs/examples/va-hover/Disabled.vue [hunk 5]
```diff
@@ -0,0 +1,20 @@
+<template>
+  <div>
+    <va-hover disabled>
+      <div style="background-color: #babfc2;">
+        {{ value }}
```
Comment (asvae) [inline packages/docs/examples/va-hover/Disabled.vue:5]: Also not sure, what you're trying to achieve here.

File: packages/docs/components/page-configs/va-hover/api-options.ts [hunk 6]
```diff
@@ -0,0 +1,20 @@
+import { ManualApiOptions } from 'vuestic-ui/src/services/api-docs/ManualApiOptions'
```
Comment (asvae) [inline packages/docs/components/page-configs/va-hover/api-options.ts:1]: Sizing, grids, see `#vuestic-docs` channel message
image

File: packages/docs/examples/va-hover/Disabled.vue (hunk 3 above)
Comment (SergeiPleshkov) [reply packages/docs/examples/va-hover/Disabled.vue:8]: To show that data binding is two way.

**Pull request merged**